WordPress/gutenberg
# DataViews: rename `InFilter` component to `FilterSummary`

Part of https://github.com/WordPress/gutenberg/issues/55083

## What?

Renames the `InFilter` component to `FilterSummary`.

## Why?

Because that's a better representation of that the React component does. It also frees the name space for a new `InFilter` subcomponent to be shared across all the places it's used (top-level, column, add filter).

## How?

- Renames the `in-filter.js` file to `filter-summary.js`
- Renames the `InFilter` component to `FilterSummary`.

## Testing Instructions

- Enable the "view admin" experiment and visit "Appareance > Editor > Pages > Manage all pages".
- Verify the filters still work as expected.
